client/ipa-getkeytab.c

```diff
@@ -559,34 +559,48 @@ static int ldap_get_keytab(krb5_context krbctx, bool generate, char *password,
     return ret;
 }
 
-static char *ask_password(krb5_context krbctx)
+/* Prompt for either a password.
+ * This can be either asking for a new or existing password.
+ *
+ * To set a new password provide values for both prompt1 and prompt2 and
+ * set match=1 to enforce that the two entered passwords match.
```

Comment:
bool, so `match = true`

Reply:
Gah, that's what I get for flipping it from an int to a bool at the last minute.